crates/node/core/src/args/network.rs

```diff
@@ -50,7 +50,7 @@ pub struct NetworkArgs {
     ///
     /// Will fall back to a network-specific default if not specified.
     #[arg(long, value_delimiter = ',')]
-    pub bootnodes: Option<Vec<TrustedPeer>>,
+    pub bootnodes: Option<Vec<NodeRecord>>,
```

Comment:
does this matter @Rjected ?

Reply:
>  @Rjected not sure if we need trustedpeer format and resolve here?

We actually do need trusted peer format here, since CLI is one of the only places where users will use domains in enodes